### DISCUSSION TITLE
可以添加elementui吗

### DISCUSSION THREAD

**Discussion by lonely-LXC** [DefinitelyTyped/DefinitelyTyped#59393] [Request a new @types package]
可以添加elementui吗，还是说我没有找到

**Comment by zhoushengdao** [comment]
[element-ui](https://www.npmjs.com/package/element-ui) 是自带 typescript 类型定义的，所以说直接安装就可以了，不需要下载别的了。
![image](https://user-images.githubusercontent.com/53334669/177041260-f27229c5-18ea-47af-bb0f-aba15b8c2012.png)
![image](https://user-images.githubusercontent.com/53334669/177041363-f81d5607-1cf0-4592-97d5-e480a88680ae.png)
